benchmark/README.md

```diff
@@ -0,0 +1,8 @@
+Ghostferry benchmark setup
```

Comment:
Ia this benchmark setup specific to linux environment, and if yes we should mention it in the README